contracts/AssetToken.sol

```diff
@@ -16,6 +16,9 @@ contract AssetToken is IERC777, Initializable {
     using SafeMath for uint256;
     using Address for address;
 
+    //if negative a whitelist is assumed
+    bool private _isUsingBlacklist;
```

Comment:
I wonder if people will think setting _isUsingBlacklist to false will just mean no blacklisting, not that whitelisting is enabled